src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const ROBOT_VALUES = Object.values(robot);
+
+  function findDuplicates(arry) {
+    return arry.filter((item, index) => arry.indexOf(item) !== index);
+  };
+
+  const DUP_VALUES = findDuplicates(ROBOT_VALUES);
```

Comment:
Use 'const duplicateValues' instead of 'const DUP_VALUES' to follow camelCase naming conventions.